README.md

```diff
@@ -1,9 +1,108 @@
+[![GitHub License](https://img.shields.io/github/license/canoical/edgex-checkbox-provider)](https://choosealicense.com/licenses/apache-2.0/) 
+[![GitHub Pull Requests](https://img.shields.io/github/issues-pr-raw/canonical/edgex-checkbox-provider)](https://github.com/edgexfoundry/edgex-cli/pulls) 
+[![GitHub Contributors](https://img.shields.io/github/contributors/canonical/edgex-checkbox-provider)](https://github.com/edgexfoundry/edgex-cli/contributors) 
+[![GitHub Committers](https://img.shields.io/badge/team-committers-green)](https://github.com/orgs/canonical/teams/edgex/members) 
+[![GitHub Commit Activity](https://img.shields.io/github/commit-activity/m/edgexfoundry/edgex-cli)](https://github.com/canonical/edgex-checkbox-provider/commits)
 
-The upstream repository is hosted on Github:  
-https://github.com/canonical/edgex-checkbox-provider
+## Introduction
 
-The tests run via the Launchpad mirror at:  
-https://code.launchpad.net/checkbox-provider-edgex
+This project contains the checkbox tests of the [edgex foundry](https://docs.edgexfoundry.org/) snaps. 
+[Checkbox](https://checkbox.readthedocs.io/en/latest/) is a test automation software performed by the Canonical certification team. 
+The [upstream repository](https://github.com/canonical/edgex-checkbox-provider) is hosted on Github; 
+The tests run via the [Launchpad mirror](https://code.launchpad.net/checkbox-provider-edgex).
+
+When edgexfoundry snap is released to the latest/beta channel, the corresponding checkbox test will be triggered. 
+
+## Hardware
+
+This table shows hardware used for edgexfoundry snap checkbox tests:
+| Architecture | amd64              | amd64                          | amd64                  | arm64                         | arm64                       | arm64            | arm64                                     |           arm64           |
+| ------------ | ------------------ | ------------------------------ | ---------------------- | ----------------------------- | --------------------------- | ---------------- | ----------------------------------------- | :-----------------------: |
+| Model        | Tulip 17 BDW       | Caracalla                      | St. Louis              | RPI4B4G                       | RPI4B8G                     | RPI400           | CM4L                                      |        DragonBoard        |
+| Information  | Dell Inspiron 5758 | Dell Edge Gateway 3003 [Media] | Dell Edge Gateway 5000 | armv7l Raspberry Pi 4 Model B | Raspberry  Pi 4 Model B 8GB | Raspberry Pi 400 | Raspberry Pi Compute Module 4 Lite - CM4L | Qualcomm DragonBoard 410c |
+| Ubuntu OS    | Core 16/16 LTS     | Core 16                        | Core 16                | 16 LTS                        | 16 LTS                      | 16 LTS           | 16 LTS                                    |          16 LTS           |
+
+
+
+## Usage
+
+### Run tests using checkbox CLI
+```bash
+git clone https://github.com/canonical/edgex-checkbox-provider.git
```

Comment:
This isn't necessary. It would be invalid in forks and github already has this.

Reply:
Recommand